mridata-3dfse-knee/unet/ssdu/SSDU_Aug_UNET_13_Scan.yaml

```diff
@@ -0,0 +1,58 @@
+# Inherit fields from the unet.yaml config.
+_BASE_: "../../templates/unet.yaml"
+DATALOADER:
+  SUBSAMPLE_TRAIN:
+    # 1 supervised and 13 unsupervised scans for training
+    NUM_TOTAL: 14
+    NUM_UNDERSAMPLED: 13
+# Specify an output directory that is unique for this experiment. 
+OUTPUT_DIR: results://vortex/mridata_knee_3dfse/unet/ssdu/ssdu_aug_unet_13_scan
+DESCRIPTION:
+  # Use f-strings to specify reading values in braces ({}) the config file.
+  # This can only be done for string fields.
+  BRIEF: f"SSDU Aug U-Net {AUG_TRAIN.UNDERSAMPLE.ACCELERATIONS}x mridata - {MODEL.SSDU.MASKER.PARAMS.kind}/loss={MODEL.RECON_LOSS.NAME}"
+  EXP_NAME: vortex/mridata_knee_3dfse/unet/SSDU_Aug
+  TAGS:
+  - baseline
+  - ssdu
+  - 16x
+  - mridata_knee_3dfse
+  - unet
+# For motion aug, A2R and Consistency were kept with std_dev range of 0.2 to 0.5.
+MODEL:
+  A2R:
+    META_ARCHITECTURE: UnetModel 
+    USE_SUPERVISED_CONSISTENCY: false  # TUNE 
+  CONSISTENCY:
+    AUG:
+      MRI_RECON:
+        AUG_SENSITIVITY_MAPS: true
+        SCHEDULER_P:
+          IGNORE: false
+        TRANSFORMS:
+        - name: RandomMRIMotion 
+          p: 0.2
+          std_devs:
+          - 0.2
+          - 0.5
```

Comment:
This configuration is for the 1D motion augmentation used in the original VORTEX paper. This is a good place to start, but in the future we will need to create a new config with 2D roto-translational motion